connection.go

```diff
@@ -14,11 +14,17 @@ type Connection interface {
 	Remote() *Peer
 
 	getLocal() *Peer
-	breakTie(Connection) connectionTieBreak
 	remoteTCPAddress() string
 	isOutbound() bool
 	isEstablished() bool
+}
+
+type LConnection interface {
```

Comment:
I propose `tiebreakConnection` or `internalConnection`. (As it's only used in package mesh, it should not be exported.)